crates/ruff/src/rules/ruff/snapshots/ruff__rules__ruff__tests__RUF017_RUF017.py.snap

```diff
@@ -0,0 +1,53 @@
+---
+source: crates/ruff/src/rules/ruff/mod.rs
+---
+RUF017.py:5:1: RUF017 [*] Avoid quadratic list summation
+  |
+4 | # RUF017
+5 | sum([x, y], start=[])
+  | ^^^^^^^^^^^^^^^^^^^^^ RUF017
+6 | sum([x, y], [])
+7 | sum([[1, 2, 3], [4, 5, 6]], start=[])
+  |
+  = help: Replace with `functools.reduce`
+
+ℹ Suggested fix
+  1 |+import functools
+  2 |+import operator
+1 3 | x = [1, 2, 3]
+2 4 | y = [4, 5, 6]
+3 5 | 
+4 6 | # RUF017
+5   |-sum([x, y], start=[])
+  7 |+functools.reduce(operator.iadd, [x, y], [])
+6 8 | sum([x, y], [])
+7 9 | sum([[1, 2, 3], [4, 5, 6]], start=[])
+8 10 | sum([[1, 2, 3], [4, 5, 6]], [])
+
+RUF017.py:7:1: RUF017 [*] Avoid quadratic list summation
+  |
+5 | sum([x, y], start=[])
+6 | sum([x, y], [])
+7 | sum([[1, 2, 3], [4, 5, 6]], start=[])
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ RUF017
+8 | sum([[1, 2, 3], [4, 5, 6]], [])
```

Comment:
Is it intentional this only complains when start is passed by kwarg?

Reply:
Just my mistake, @evanrittenhouse had it right initially. I looked at the docs and saw the `/` for the positional-only `iterable` argument, and my mind immediately went to keyword-only, wasn't thinking straight. PR welcome, or I'll fix a little later.

Reply:
Thanks both! Easy fix, will open a PR

Reply:
https://github.com/astral-sh/ruff/pull/6664